sickbeard/postProcessor.py

```diff
@@ -1061,7 +1061,7 @@ def process(self):  # pylint: disable=too-many-return-statements, too-many-local
         for cur_ep in [ep_obj] + ep_obj.related_episodes:
             try:
                 self._delete(cur_ep.location, associated_files=True)
-
+                sickbeard.KODI_LIBRARY_CLEAN_PENDING = True
```

Comment:
its a global var. not config var. In __init__ it start as True.

So when we call the self_delete it delete the old file then I set to clean library